engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -369,13 +387,40 @@ def scc_deactivate_product
           auth = request.headers['HTTP_AUTHORIZATION']
           if @system.byos? && @product[:product_type] != 'base'
             response = SccProxy.deactivate_product_scc(auth, @product, @system.system_token)
-            unless response.code_type == Net::HTTPOK
-              error = JSON.parse(response.body)
-              error['error'] = SccProxy.parse_error(error['error'], params[:token], params[:email]) if error['error'].include? 'json'
-              logger.info "Could not de-activate product '#{@product.friendly_name}', error: #{error['error']} #{response.code}"
-              raise ActionController::TranslatedError.new(error['error'])
+            handle_response(response)
+          elsif @system.hybrid? && @product.extension?
+            # check if product is on SCC and
+            # if it is -> de-activate it
+            scc_systems_activations = find_hybrid_product_on_scc(headers)
```

Comment:
For me, this call to SCC fails with 'Unauthorized 401'. I assume that is, because the headers here are generated without providing the `Authorization: ` header from the original request, and the instance id.